daliuge-engine/test/apps/test_simple.py

```diff
@@ -135,6 +135,22 @@ def test_helloworldapp(self):
         h.execute()
         self.assertEqual(six.b(h.greeting), droputils.allDropContents(b))
 
+    def test_ngasio(self):
+        nd_in = NgasDROP('HelloWorld.txt', 'HelloWorld.txt')
+        nd_in.ngasSrv = 'ngas.ddns.net'
```

Comment:
I guess this is fine for now, but if possible it'd be better to remove this dependency on an external service, as it can cause sporadic test failures. A better alternative would be to run the test against a local NGAS installation, but only if one exists (using `@unittest.skipIf` or similar).